ts/components/conversation/composition/CompositionBox.tsx

```diff
@@ -521,8 +522,7 @@ class CompositionBoxInner extends Component<Props, State> {
 
     const allMembers = allPubKeys.map(pubKey => {
       const convo = ConvoHub.use().get(pubKey);
-      const profileName =
-        convo?.getNicknameOrRealUsernameOrPlaceholder() || window.i18n('anonymous');
+      const profileName = convo?.getNicknameOrRealUsernameOrPlaceholder() || PubKey.shorten(pubKey);
```

Comment:
Are you happy to leave shortening the pubkey outside of `getNicknameOrRealUsernameOrPlaceholder()`?

Reply:
yeah, `getNicknameOrRealUsernameOrPlaceholder` always returns a `string`, 
but ?. makes it sometimes `undefined`, and we need a fallback for that case.

Reply:
cool resolving